queries.sql

```diff
@@ -205,6 +205,12 @@ GROUP BY s.specie_name
 ORDER BY num_owned DESC
 LIMIT 1;
 
-
-
-
+-- QUERIS TO CHECK PERFORMANCE
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+-- queries for optimizing the visits and owners tables
+CREATE INDEX idx_visits_animal_id ON visits(animal_id);
+CREATE INDEX idx_visits_vet_id ON visits(vet_id);
+CREATE INDEX idx_owners_email ON owners(email);
```

Comment:
- You have done great so far :+1: :100: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 

- [ ] You have done well using INDEX to improve the execution time of your query, but these additions of indexes should be done on the `schema.sql` file and not in the queries.sql. :+1: :+1: